src/tictactoe/Tictactoe.java

```diff
@@ -0,0 +1,54 @@
+package tictactoe;
+
+import java.util.Scanner;
+
+//Написать игру крестики нолики (двумерный массив размерностью 3х3).
+//1. Не используя классы и методы - все в методе main
+//2. Не используя утильные классы (Arrays и Math)
+//3. Игра должна продолжаться до тех пор, пока не будет выявлен победитель
+// (три "О" или три "Х" в ряд или по диагонали) или пока не закончатся свободные клетки
+public class Tictactoe {
```

Comment:
It works but of course I want enter O after X)) Now I can set multiple X in the row)